addon/templates/components/validated-input.hbs

```diff
@@ -1,8 +1,9 @@
 {{#if (not-eq type "checkbox")}}
-  {{#if label}}
-      <label class={{config.css.label}} for={{inputId}}>
-        {{label}}{{#if required}} {{requiredLabel}}{{/if}}
-      </label>
+  {{#if (or label labelComponent)}}
+    <span style="display: inline-block;">
+      {{component (if labelComponent labelComponent "validated-label")
+        isLabel=true label=label config=config inputId=inputId required=required}}
```

Comment:
I'd prefer always using `{{component labelComponent}}` and just setting the default value of `labelComponent` to `'validated-label'`